src/nupic/algorithms/SDRClassifier.hpp

```diff
@@ -34,7 +34,11 @@
 
 #include <nupic/types/Types.hpp>
 #include <nupic/types/Sdr.hpp>
+
 #include <nupic/types/Serializable.hpp>
+#include <cereal/types/vector.hpp>
+#include <cereal/types/deque.hpp>
+#include <cereal/types/map.hpp>
```

Comment:
make these included in Serializable (vector already is. does cereal.hpp not import all of them?), so the implementing class does not have to do anything than Serializable.hpp

Reply:
Yes, those includes should be in Serializable.hpp